js/src/dom/selector-engine.js

```diff
@@ -26,10 +26,10 @@ const getSelector = element => {
       hrefAttribute = `#${hrefAttribute.split('#')[1]}`
     }
 
-    selector = hrefAttribute && hrefAttribute !== '#' ? parseSelector(hrefAttribute.trim()) : null
+    selector = hrefAttribute && hrefAttribute !== '#' ? hrefAttribute.trim() : null
   }
 
-  return selector
+  return selector?.split(',').map(sel => parseSelector(sel)).join(',')
```

Comment:
I wouldn't use optional chaining yet since IIRC we don't use it anywhere yet. We could switch at some point, not sure when, though :)